module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,19 @@
+#include "sort.hpp"
+
+bool stringCompare(const std::string& first, const std::string& second) {
+    size_t first_len = first.length();
+    size_t second_len = second.length();
+    if (first_len == second_len) {
+        return first < second;
+    }
+    return first_len < second_len;
+}
+
+/* 
+   I really think we need copy here, just because Command Query separation principle.
+*/
```

Comment:
Maybe resolve all conflicts and remove comment, before push :)?

Reply:
Done. I was mine internal (between me & me) conflict ;d